projects/Mallard/src/screens/settings/help-screen.tsx

```diff
@@ -25,6 +29,32 @@ const HelpScreen = ({ navigation }: NavigationInjectedProps) => {
                         },
                     ]}
                 />
+                <Heading>Contact us</Heading>
+                <List
+                    onPress={({ onPress }) => onPress()}
+                    data={[
+                        createSupportMailTo(
+                            'Report an issue',
+                            'apps.feedback@theguardian.com',
```

Comment:
Same for these emails - can move to words.ts or similar.

Reply:
Thanks, done 🆗